Assets/com.gdcbd.bossbattle/Scripts/Player/PlayerHealth.cs

```diff
@@ -0,0 +1,62 @@
+using System;
+using UnityEngine;
+using UnityEngine.Serialization;
+
+namespace com.gdcbd.bossbattle.player
+{
+    [CreateAssetMenu(fileName = "New Player Health", menuName = "BossBattle/PlayerHealth", order = 0)]
+    public class PlayerHealth : ScriptableObject
```

Comment:
Wrong name. It's called player health but stores lots of data that's not realated to health